Formula/ghostscript.rb

```diff
@@ -69,38 +63,17 @@ class Ghostscript < Formula
   end
 
   def install
-    # Fix vendored tesseract build error: 'cstring' file not found
-    # Remove when possible to link to system tesseract
-    ENV.append_to_cflags "-stdlib=libc++" if ENV.compiler == :clang
-
-    # Fix VERSION file incorrectly included as C++20 <version> header
-    # Remove when possible to link to system tesseract
-    rm "tesseract/VERSION"
-
     # Delete local vendored sources so build uses system dependencies
-    rm_rf "expat"
-    rm_rf "freetype"
-    rm_rf "jbig2dec"
-    rm_rf "jpeg"
-    rm_rf "lcms2mt"
-    rm_rf "libpng"
-    rm_rf "openjpeg"
-    rm_rf "tiff"
-    rm_rf "zlib"
-
-    args = %w[
-      --disable-compile-inits
-      --disable-cups
-      --disable-gtk
-      --with-system-libtiff
-      --without-x
-    ]
-
-    if build.head?
-      system "./autogen.sh", *std_configure_args, *args
-    else
-      system "./configure", *std_configure_args, *args
-    end
+    libs = %w[expat freetype jbig2dec jpeg lcms2mt libpng openjpeg tiff zlib]
```

Comment:
No system `tesseract` or `leptonica`? Maybe we could just manually pass the right `-I`, `-L` and `-l` flags directly to `configure` or `make`.

Reply:
Configure script will check for local source code and auto-disable feature if it doesn't exist https://git.ghostscript.com/?p=ghostpdl.git;a=blob;f=configure.ac;h=bb578250c2a3ba40c3479e57fd59e07f2c52e968;hb=f9673cbc87df8331c5b3928d91fde25d0f111e9c#l981

---

Some Linux repositories just disable OCR support (Arch, Fedora, etc.)

Reply:
Ok, let me see about filing an issue upstream then.

Reply:
Oh, someone beat us to it by a month: https://bugs.ghostscript.com/show_bug.cgi?id=705764